store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -396,7 +396,7 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
         case RowDataUpdated(ops) =>
           val loader = dataLoader.getOrElse(prevettedLoader(pgu, truthCopyInfo))
           RowDataUpdatedHandler(loader, ops)
-          (rebuildIndex, true, truthCopyInfo, Some(loader))
+          (rebuildIndex, refreshRollup || ops.headOption.isDefined, truthCopyInfo, Some(loader))
         case LastModifiedChanged(lastModified) =>
           pgu.datasetMapWriter.updateLastModified(truthCopyInfo, lastModified)
           (rebuildIndex, true, truthCopyInfo, dataLoader)
```

Comment:
This will happen on every single update a secondary gets, so the `true` here should probably be `refreshRollup` too.

Reply:
..actually thinking about it, this change is the _only_ important one.  An empty changeset will produce _only_ a LastModifiedChange event, no rows at all (though the other change, or-ing in `nonEmpty` makes sense and I think it should happen)